aat/src/aat/resources/features/F-100 TTL Increment in Case Event/S-100.4.td.json

```diff
@@ -0,0 +1,30 @@
+{
+  "_guid_": "S-100.4",
+  "_extends_": "Import_Base_data",
+  "title": "Must successfully import a definition file where TTLIncrement column is missing",
+  "specs": [
+    "an active profile in CCD",
+    "contains missing TTLIncrement column"
+  ],
+  "request": {
+    "body": {
+      "arrayInMap": [
+        {
+          "key": "file",
+          "filePath": "uk/gov/hmcts/ccd/test_definitions/invalid/BEFTA_Master_Definition_missing_TTL_Increment_Column.xlsx"
```

Comment:
No we cannot have a successfull import test of a different version of the BEFTA_MASTER file for this test as it would replace the common one.  So we should delete this file from hmcts/ccd-test-definitions#33  (NB: the invalid folder should only contain invalid def files).  And we can test using one of the other valid def files that is missing the column.